fleet/api/http/metrics.go

```diff
@@ -126,12 +133,11 @@ func (m metricsMiddleware) ViewAgentGroupByIDInternal(ctx context.Context, group
 		labels := []string{
 			"method", "viewAgentGroupByIDInternal",
 			"owner_id", ownerID,
-			"service", "fleet",
-			"id", groupID,
+			"thing_id", groupID,
```

Comment:
here's the example of a groupID - we can't put this into thing_id, it needs its own.